templates/faq/translation_status.html

```diff
@@ -0,0 +1,96 @@
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, viewport-fit=cover">
+    <style>
+    table.status {
+        border-collapse: collapse;
+    }
+    table.status tr.header {
+        text-align: center;
+        font-weight: bold;
+    }
+    table.status tr.header td {
+        background: #ddd;
+    }
+    table.status tr.faq_category td {
+        background: #eaecef;
+    }
+    table.status td.faq-outdated {
+        background: #FFCCCC;
+    }
+    table.status td.faq-no-date {
+        background: #FFEECC;
+    }
+    table.status tr td {
+        border: 1px solid gray;
+    }
+    table.status .updated-label {
+        display: block;
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 8pt;
+        text-align: end;
+        margin-top: 4px;
+    }
+    </style>
+    <title>F.A.Q. translation status</title>
+
+</head>
+<body>
+
+{% set all_categories = get_taxonomy(kind='faq', lang = 'en') %} {# Use English as list of all FAQ categories and questions #}
+{% set faq_languages = ["ru", "de", "fr", "it", "pl", "pt", "pt-BR", "tr", "uk", "zh-Hans"] %} {# Predefined list of translation languages #}
```

Comment:
Can the list be retrieved dynamically?

Reply:
I can't find how to implement this with Zola templates.

Reply:
ACK, we'll come to it later, now manual editing is good enough.